composer.json

```diff
@@ -16,15 +16,17 @@
     ],
     "minimum-stability": "dev",
     "require": {
-        "jackalope/jackalope-transport": "*",
```

Comment:
this virtual package requirement should not have been removed. fixing in 35bd2ec12c4cfc76d3f4d0676c01fa4d4aee6621